homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -61,24 +63,38 @@ async def async_setup_platform(
             [AFSAPIDevice(name, afsapi)],
             True,
         )
-        return
 
-    host = config.get(CONF_HOST)
-    port = config.get(CONF_PORT)
-    password = config.get(CONF_PASSWORD)
-    name = config.get(CONF_NAME)
+    ir.async_create_issue(
+        hass,
+        DOMAIN,
+        "remove_yaml",
+        is_fixable=False,
```

Comment:
You should add a dealine here.
```suggestion
        breaks_in_ha_version="2023.6.0",
        is_fixable=False,
```